internal/code/parser.go

```diff
@@ -0,0 +1,74 @@
+package code
```

Comment:
All this is quite specific to cypress, right? Perhaps this `code` package should move under `cypress`?

Reply:
Good point!